examples/gno.land/p/demo/ownable/ownable.gno

```diff
@@ -40,18 +52,32 @@ func (o *Ownable) DropOwnership() error {
 		return err
 	}
 
+	prevOwner := o.owner
 	o.owner = ""
+
+	std.Emit(
+		OwnershipTransferEvent,
```

Comment:
I think we can remove the `Event` word from the name of the event - and we should think about the field name in the event when it's outputted, maybe it should be "name" instead of "type".

Reply:
The API should be `std.Emit(evt)`, which uses the type name as the "type" value and automatically computes fields from the provided event struct, instead of requiring key-value strings.

In other words, the "type" field is appropriate, but it shouldn't be an arbitrary string.